pkg/network/dhcp/configurator.go

```diff
@@ -11,69 +11,44 @@ import (
 
 const defaultDHCPStartedDirectory = "/var/run/kubevirt-private"
 
-type Configurator struct {
+type Configurator interface {
```

Comment:
Oh it took me a while to fully understand this (the interface is exported, the implementation un-exported). Looks good.